students/sanghyun/users/views.py

```diff
@@ -46,11 +47,16 @@ def post(self, request):
             data        = json.loads(request.body)
             email       = data['email']
             password    = data['password']
+            user_info   = User.objects.get(email=email)
 
-            if not User.objects.filter(email=email, password=password).exists():
-                return JsonResponse({"message": "INVALID_USER"}, status=401)
+            if bcrypt.checkpw(password.encode("utf-8"), user_info.password.encode("utf-8")):
+                token = jwt.encode({'id': user_info.id}, SECRET_KEY, algorithm = ALGORITHM)
+                return JsonResponse({'MESSAGE' : 'TOKEN_GENERATE_SUCCESS', 'token' : token}, status = 200)
+
+            return JsonResponse({"MESSAGE": "CHECK_YOUR_ID/PASSWORD"}, status = 401)
```

Comment:
if not 활용해서 아래와 같이 하실 수도 있습니다!

```suggestion
            if not bcrypt.checkpw(password.encode("utf-8"), user_info.password.encode("utf-8")):
                return JsonResponse({"MESSAGE": "CHECK_YOUR_PASSWORD"}, status = 400)
                
            token = jwt.encode({'id': user_info.id}, SECRET_KEY, algorithm = ALGORITHM)
            return JsonResponse({'MESSAGE' : 'LOGIN_SUCCESS', 'TOKEN' : token}, status = 200)

            
```

Reply:
다음번엔 try와 if, if not의 차이를 고려해서 코드 작성해 보도록 하겠습니다! 감사합니다~